lesson12/index.js

```diff
@@ -0,0 +1,63 @@
+const ul = document.getElementById('js-lists');
+const fetchButton = document.getElementById('fetchButton');
```

Comment:
```suggestion
const fetchButton = document.getElementById('js-fetchButton');
```

Reply:
Thank you for your reviewing!
I created the button in js, so I erased the id name.
Next time, I will add `js` to the head to make it easier to understand.